docs/automated-migration.md

```diff
@@ -10,50 +10,72 @@ Robolectric provides an automated migration tool to help keep your test suite up
 
-2. [Configure your project](https://errorprone.info/docs/installation) to compile using Error Prone. Quick config for Gradle (usually in `app/build.gradle`):
+2. [Configure your project](https://errorprone.info/docs/installation) to integrate Error Prone. Quick config for Gradle (usually in your module's `build.gradle`/`build.gradle.kts` file):
+
+=== "Groovy"
 
     ```groovy
     plugins {
-        id "net.ltgt.errorprone" version "0.6" apply false
+        id "net.ltgt.errorprone" version "<error_prone_plugin_version>" apply false
     }
 
-    String roboMigration = System.getenv("ROBOLECTRIC_MIGRATION")
-    if (roboMigration) {
+    String robolectricMigrations = System.getenv("ROBOLECTRIC_MIGRATIONS")
+    if (robolectricMigrations) {
         apply plugin: "net.ltgt.errorprone"
 
         dependencies {
-            errorprone "com.google.errorprone:error_prone_core:2.3.2"
-            errorproneJavac "com.google.errorprone:javac:9+181-r4173-1"
-
+            errorprone "com.google.errorprone:error_prone_core:<error_prone_version>"
             errorprone "org.robolectric:errorprone:{{ robolectric.version.current }}"
         }
 
-        afterEvaluate {
-            tasks.withType(JavaCompile) { t ->
-                options.errorprone.errorproneArgs += [
-                        '-XepPatchChecks:' + roboMigration,
-                        '-XepPatchLocation:IN_PLACE',
-                ]
-            }
+         tasks.withType(JavaCompile).configureEach {
+             options.errorprone.errorproneArgs = [
+                  '-XepPatchChecks:' + robolectricMigrations,
+                  '-XepPatchLocation:IN_PLACE',
+             ]
+         }
+    }
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    plugins {
+        id("net.ltgt.errorprone") version "<error_prone_plugin_version>" apply false
+    }
+
+    val robolectricMigrations = System.getenv("ROBOLECTRIC_MIGRATIONS")
+    if (!robolectricMigrations.isNullOrEmpty()) {
+        pluginManager.apply("net.ltgt.errorprone")
+
+        dependencies {
+            errorprone("com.google.errorprone:error_prone_core:<error_prone_version>")
+            errorprone("org.robolectric:errorprone:{{ robolectric.version.current }}")
         }
+
+         tasks.withType<JavaCompile>().configureEach {
+             options.errorprone.errorproneArgs = listOf(
+                  "-XepPatchChecks:$robolectricMigrations",
+                  "-XepPatchLocation:IN_PLACE",
+             )
+         }
     }
     ```
 
     You don't need to commit this change.
 
-3. Run the migrations. Due to limitations of Error Prone, you'll need to manually do this in a couple steps:
+3. Run the migrations:
 
     ```bash
-    ROBOLECTRIC_MIGRATION=DeprecatedMethods ./gradlew clean :compileDebugUnitTestJava
-    ROBOLECTRIC_MIGRATION=ShadowUsageCheck ./gradlew clean :compileDebugUnitTestJava
+    ROBOLECTRIC_MIGRATION=DeprecatedMethods,ShadowUsageCheck ./gradlew clean :compileDebugUnitTestJava
```

Comment:
It seems that it is now possible to pass multiple checks at once. See https://errorprone.info/docs/patching
<img width="963" alt="Screenshot 2024-09-22 at 11 46 12" src="https://github.com/user-attachments/assets/95af57ed-6417-4913-a4da-25c961d4cdf8">